lib/matplotlib/backend_bases.py

```diff
@@ -1528,7 +1528,10 @@ def __init__(self, name, canvas, x, y, guiEvent=None):
             axes_list = [self.canvas.mouse_grabber]
 
         if axes_list:  # Use highest zorder.
-            self.inaxes = max(axes_list, key=lambda x: x.zorder)
```

Comment:
Can we do `max(reversed(axes_list), key=lambda x: x.zorder)` instead?

Reply:
I'd rather be explicit about the fact that order in axes_list *does* matter (at least I wouldn't have thought that the way `max` breaks ties is so important by skimming through the code).